main.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'my_list'
+
+list = MyList.new([1, 2, 3, 4])
```

Comment:
- [ ] ist = MyList.new(1, 2, 3, 4), KIndly remove the array & pass in initialize.